packages/hemera-mongo-store/store.js

```diff
@@ -17,9 +28,16 @@ class MongoStore extends Store {
    *
    * @memberOf MongoStore
    */
-  constructor (driver, options = {}) {
-    options.mongo = {}
+  constructor (driver,options = { mongo: {}, store: {} }) {
     super(driver, options)
+    if (!this.options.hasOwnProperty('store')) {
```

Comment:
Yes, but the default option can be overwritten with a `options` object without `store`... so could be `unefined`